client/src/components/Verejne/GoogleMap/Marker/Marker.scss

```diff
@@ -25,14 +25,30 @@
 }
 
 .ClusterMarker {
-    border-radius: $marker-height;
+    border-radius: 50%;
     border: 3px solid $cluster-marker-border;
     background-color: $marker-color;
     text-align: center;
     color: $marker-color !important;
 }
 
 .SimpleMarker {
-    background: url('./mapIcon.svg') no-repeat;
+    background: url('../../mapIcon.svg') no-repeat;
     background-size: contain;
 }
+
+.CompanyMarker {
+    display: flex;
+    svg {
+        width: 50%;
+        height: 50%;
+        display: flex;
+        margin: auto;
+    }
+    &__Normal svg {
+        color: $marker-color !important;
```

Comment:
you are using quite a lot of importants in css

my assumption is there are some hard to override defaults on the google maps component, so it's probably the only way, but still - think over other options (adding another custom class?), because this can get hard to maintain